src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +129,14 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts the persons in the list in alphabetical order and returns itself.
+     */
+    public UniquePersonList sort() {
+        internalList.sort(new Person.NameComparator());
```

Comment:
Try not to modify the original data, instead, modify a copy (: This gives more freedom to keep track of some properties of the original `UniquePersonList` and implement more sort commands in the future - e.g. sort by recently added